include/itkAttenuationImageFilter.h

```diff
@@ -82,33 +82,75 @@ class ITK_TEMPLATE_EXPORT AttenuationImageFilter : public ImageToImageFilter<TIn
   itkSetInputMacro(MaskImage, TMaskImage);
   itkGetInputMacro(MaskImage, TMaskImage);
 
-  itkGetConstMacro(LabelValue, MaskPixelType);
+  /** Label value indicating pixel should be included in analysis.
```

Comment:
indicating pixel should be included -> indicating which mask pixel values should be included